spec/cocoapods/command/gen_spec.rb

```diff
@@ -304,6 +304,27 @@
             )
           end
         end
+
+        describe 'when specifying :podspec_paths in configuration files and CLI args' do
+          let(:argv) { %w[C.podspec D.podspec] }
```

Comment:
what happens if the podspecs in ARGV don't completely match what's in the local config?

Reply:
🤦‍♂ too many `A B C D`.. will update the test to make it clearer. The answer is the CLI args win and override the local config, as mentioned in the PR discussion

Reply:
👍🏻